### ISSUE TITLE
Example site is down

### ISSUE THREAD

**Issue by josephg** [Ircam-RnD/binauralFIR#2]
The [example site](https://ircam-rnd.github.io/binauralFIR/) listed in the project description isn't working. I see this in the console:

```
Failed to load resource: the server responded with a status of 404 (Not Found) https://rawgit.com/Ircam-RnD/binauralFIR/master/binauralfir.min.js
Failed to load resource: the server responded with a status of 404 (Not Found) https://rawgit.com/ircam-rnd/buffer-loader/master/buffer-loader.min.js
Failed to load resource: the server responded with a status of 404 (Not Found) https://rawgit.com/ircam-rnd/player/master/player.min.js
Uncaught ReferenceError: createBufferLoader is not defined (index):48
```

**Comment by ouhouhsami** [comment]
Well, sorry for that. We are in large refactoring.
You should have a *non* working example here : http://jsfiddle.net/ouhouhsami/4ptmsc5d/  but which should be a bit clearer on how the lib works.
Otherwise, in a month, we should have a cleaner repo.

**Comment by josephg** [comment]
Cool - mostly I wanted to know what to expect it to sound like before I started messing around with the library. I eventually got the examples working, although I also had to recompile loaders.js (it was missing require somehow). Also I uglified the hrtf database file, which dropped it down to 6mb (2.5mb gzipped) - still a fair bit bigger than I'd like, but better than it was.

Its uploading to dropbox [here](https://dl.dropboxusercontent.com/u/2494815/hrtf/index.html) if anyone wants to play with it, although it'll take a few more minutes to upload.
